### ISSUE TITLE
staticFriction for HingleJoint doesn't work

### ISSUE THREAD

**Issue by kidsjtu** [cyberbotics/webots#2483]
**Describe the Bug**

**Steps to Reproduce**
1. When conducting simulation of manipulators, different "staticFriction" values of a HingleJoint lead to the same motor torques.
2. As shown in the screenshot, staticFriction is set to be 0.5, but the max value of the motor torques is smaller than 0.1.

**Expected behavior**
Fix it.

**Screenshots**
![image](https://user-images.githubusercontent.com/16715235/100608391-48ae2900-3347-11eb-935c-84a44f568e0e.png)

**System**
 - Operating System: Windows 10
 - Graphics Card: NVIDIA GeForce 1060 3 GB

**Additional context**
no

**Comment by lkx8421** [comment]
Is fixed? @omichel

**Comment by omichel** [comment]
I don't think so. Did you try the latest version of Webots to see if the bug is still there?
